code/game/machinery/doors/multi_tile.dm

```diff
@@ -266,6 +296,18 @@
 		unlock(TRUE)
```

Comment:
Can you also add a `if(user.action_busy)` before the do_after and the to_chat because its goofy queen can spam click this.